### PR TITLE
update script to enable establishment option

### PR DESCRIPTION
**Pull request by davidthomas22**
*updates the title of the CSV file with the establishment name
*added the argument to the script to remove the requirement for URL and token for each establishment.

### CODE REVIEW

File: moj-node/bin/download-feedback [hunk 1]
```diff
@@ -19,14 +19,29 @@ const mkdirp = require('mkdirp');
...
 const { getEnv } = require('../utils');
 
-const token = getEnv('MATOMO_TOKEN');
-const siteUrl = getEnv('MATOMO_URL');
+if (process.argv.length < 4) {
+  console.log(
+    '*****Please ensure you enter the command in the below format:*****',
+  );
+  console.log(
+    './bin/download-feedback [Start date YYYYMMDD],[End date YYYYMMDD]  [Establishment Name lowercase]',
```
Comment (romidane) [inline moj-node/bin/download-feedback]: The dates should say `YYYY-MM-DD`

Comment (romidane) [reply moj-node/bin/download-feedback]: can you change `[Start date YYYYMMDD],[End date YYYYMMDD]` to `[Date range YYYY-MM-DD,YYYY-MM-DD]` ?

Comment (davidthomas22) [reply moj-node/bin/download-feedback]: Yes agreed i can see how that may result in confusion

File: moj-node/bin/download-feedback [hunk 2]
```diff
@@ -19,14 +19,27 @@ const mkdirp = require('mkdirp');
 
 const { getEnv } = require('../utils');
 
-const token = getEnv('MATOMO_TOKEN');
-const siteUrl = getEnv('MATOMO_URL');
+if (process.argv.length < 4) {
+  console.log(
+    '*****Please ensure you enter the command in the below format:*****',
+  );
+  console.log('./bin/download-feedback [Date range: YYYY-MM-DD,YYYY-MM-DD]');
```
Comment (romidane) [inline moj-node/bin/download-feedback]: `console.log('./bin/download-feedback "YYYY-MM-DD,YYYY-MM-DD" [establishment name]);`

**Pull request merged**